R/helper.R

```diff
@@ -0,0 +1,43 @@
+#' Retrieve SingleR reference datasets from Github
+#'
+#' \code{getSinglerRef} retrieves \code{SingleR} reference datasets from Github. 
+#'
+#' @param dataset String referring to SingleR reference dataset to retrieve. 
+#'     Options include:
+#'     \itemize{
+#'       \item "hpca"
+#'       \item "blueprint_encode"
+#'       \item "immgen"
+#'       \item "mouse.rnaseq"
+#'     }
+#' @return SingleR reference dataset.
```

Comment:
This is a bit too vague; I believe the output is a list with various fields of interest? Have a look at `trainSingleR`'s `@return` field for the level of detail required; think of what a skillful newcomer would need to interpret the output.